IDE/Renesas/e2studio/RX72N/EnvisionKit_Toppers/README-en.md

```diff
@@ -0,0 +1,108 @@
+## Renesas RX72N EnvisionKit with Toppers OS
+
+
+This demo client and server run on "Toppers OS" and Renesas "RX72N EnvisionKit" with wolfSSL.
+
+
+# Prerequisite
+
+
+1.Renesas e² studio Version: 2022-07 (22.7.0) or later  
+2.Renesas e² studio BSP   
+3.Toppers OS 1.9.1 (Patch RX720N version)    
+<br>
+|element|name/version|
+|:--|:--|
+|Renesas e² studio Version|GR-2022-07 (22.7.0) or later|
+|Toppers OS|1.9.3|
+|Toppers Configurator|1.9.6|
+|Renesas BSP r_bsp|7.10|
+|Renesas BSP r_cmt_rx|5.10|
+<br>
+
+
+# Build libraries
+  # 1. Build the wolfSSL library
+   Create the Toppers library and wolfSSL library required for this demo
+
+  1-1.Press [Browse (R)] for [Select root directory (T)] in the [Import project] dialog
+  
+  1-2. Select [wolflib] from the git repository wolfssl/IDE/Renesas/e2studio/RX72N/EnvisionKit_Toppers Press [Select Folder]
+  
+  1-3. Click the created project in Project Explorer and select the [Build Project (B)] key from the pull-down menu to build.
+  (When performing the above operations, make sure that [Toolchain:] is selected in the [Project] [Properties] [Settings] [toolchain] tab.)
+  
+  1-4. [libwolflib.a] is generated in [wolflib/Debug]
+
+  # 2. Build TOPPERS library
+  
+  2-1.To build the Toppers library, download [asp-1.9.3 Renesas BSP version] from [https://www.toppers.jp/asp-d-download.html] and download [/wolfssl/ IDE/Renesas/e2studio/RX72N/EnvisionKit_Toppers/]
+  
+  2-2.To build the Toppers library, download [Configurator Release 1.9.6 (binary for Windows)] from [https://www.toppers.jp/cfg-download.html] and unzip it with [1.5] Create a [cfg/cfg] directory in the [asp] directory and extract it as [cfg.exe] inside
+  
+  2-3.Execute the shell script shown below to apply the patch.
+    (Copy the files necessary for using EDMAC)
+```
+  $ pwd [Individual installation environment]/wolfssl/IDE/Renesas/e2studio/RX72N/EnvisionKit_Toppers
+  ./setting.sh
+```
+
+  2-4. Confirmation of advance preparation
+
+    Please prepare an environment such as Msys2 in advance for command execution.
+    Install the gcc toolchain with Msys2.
+    In the Msys2 environment, it is necessary to set the path of the Renesas environment in advance.bashrc etc.
+
+--- Here is a setting example: Specify the Renesas toolchain path ---
+
+```
+$ export PATH=PATH=$PATH:\/C/ProgramData\/GCC\ for\ Renesas\ RX\ 8.3.0.202202-GNURX-ELF/rx-elf/rx-elf/bin
+```
+
+2-5. After confirming the settings, do the following
+
+```
+$ pwd [Individual installation environment]/wolfssl/IDE/Renesas/e2studio/RX72N/EnvisionKit_Toppers/asp
+$ perl ./configure -T rx72n_gcc
+$ make depend
+```
+
+  2-6. Project [Properties] → [C/C++ Build] → [Environment] dialog [Environment variable to set] [Add] button, enter [C_PROJECT] in [Name:] in [New variable] dialog , enter ${ProjDirPath} for Value:.
+
+  2-7. Click the created project in Project Explorer and select the [Build Project (B)] key from the pull-down menu to build.
```

Comment:
Need to load the project under <wolfSSL>\IDE\Renesas\e2studio\RX72N\EnvisionKit_Toppers\Toppers_app first?

Reply:
Not required if asp directory is in place

Reply:
[2-6]. mentions "Project [Properties]...". Which project should user select?

Reply:
I thought it was as noted. We will match the level of description with other projects.